### PR TITLE
Fix Help Center 

### PR DESCRIPTION
**Pull request by TPGPL**

### CODE REVIEW

File: wiki/Help_Center/en.md [hunk 1]
```diff
@@ -76,8 +74,7 @@ If you have made an account whilst being unaware of this rule somehow, cease pla
 
 ------------------------------------------------------------------------
 
-Am I allowed to change my account's username?
----------------------------------------------
+### Am I allowed to change my account's username?
 
 osu! supporters can get change their usernames, which can be [at the osu!store](http://store.ppy.sh/store/product/32).
```
Comment (MegaApplePi) [inline wiki/Help_Center/en.md]: might as well link osu!supporters
`[osu!supporters](/wiki/osu!supporter/)`

File: wiki/Help_Center/en.md [hunk 2]
```diff
@@ -124,15 +118,13 @@ If you believe that you have had your user page access revoked in error, feel fr
 
 ------------------------------------------------------------------------
 
-What are these "user pages" that I see on other players profiles?
------------------------------------------------------------------
+### What are these "user pages" that I see on other players profiles?
 
 osu!supporters are given the privilege of editing their very own page for display on their profile for the world to see. User pages may contain anything you like - so long as it follows the community rules!
```
Comment (MegaApplePi) [inline wiki/Help_Center/en.md]: ditto? I don't know if osu!supporter should be linked again

File: wiki/Help_Center/en.md [hunk 3]
```diff
@@ -124,15 +118,13 @@ If you believe that you have had your user page access revoked in error, feel fr
...
 
 ------------------------------------------------------------------------
 
-When I try to register, all the fields say "invalid request"!
--------------------------------------------------------------
+### When I try to register, all the fields say "invalid request"!
 
 This is a known issue, and we aren't yet sure what causes it. Some possible workarounds are:
```
Comment (MegaApplePi) [inline wiki/Help_Center/en.md]: Change to
`This has been a known issue; however, we aren't sure what causes this problem. If you see the described issue, try these workarounds:`

File: wiki/Help_Center/en.md [hunk 4]
```diff
@@ -144,32 +136,28 @@ If none of these solutions work for you, or the first two don't work and you hav
...
 
 Gameplay related issues
-=======================
+----------------------------
 
-My combo/scores/accuracy display are missing!
----------------------------------------------
+### My combo/scores/accuracy display are missing!
 
 A feature that toggles the HUD (scores, combo, accuracy display) was added recently.
```
Comment (MegaApplePi) [inline wiki/Help_Center/en.md]: below this line, change `**shift + tab**` to `` `Shift` + `Tab` ``

File: wiki/Help_Center/en.md [hunk 5]
```diff
@@ -179,8 +167,7 @@ If you changed the keybinding that toggles the scoreboard, press and hold shift
```
Comment (MegaApplePi) [inline wiki/Help_Center/en.md]: above this line change `shift` to `` `Shift` ``

File: wiki/Help_Center/en.md [hunk 6]
```diff
@@ -237,17 +221,15 @@ If smoke appears when you don't want it to, go to **Main Menu**, from there sele
 
 ------------------------------------------------------------------------
 
-How do I save my own replay of a completed song?
-------------------------------------------------
+### How do I save my own replay of a completed song?
 
 You can save your replay after you have completed the song, in order to do that, once you are at the post-game results screen, hit the **F2** button. That will save the replay in the **/Replays/** folder under the osu! installation directory.
```
Comment (MegaApplePi) [inline wiki/Help_Center/en.md]: Change `**F2**` to `` `F2` ``
Change `**/Replays/** to `` `Replays` `` (I think we have decided to use code blocks for folder names and directories

File: wiki/Help_Center/en.md [hunk 7]
```diff
@@ -261,15 +243,13 @@ If none of this helps, please send in a support ticket to accounts@ppy.sh with m
 
 ------------------------------------------------------------------------
 
-I can't submit my score, osu! says my game version is too old!
---------------------------------------------------------------
+### I can't submit my score, osu! says my game version is too old!
 
 If osu! doesn't update automatically, you can force update by going into **Options** and selecting **Run Updater**. Alternatively, you can close osu! and restart the osu.exe **while holding shift**. This will provide you multiple recovery options.
```
Comment (MegaApplePi) [inline wiki/Help_Center/en.md]: unbold `**while holding shift**` and change `shift` to `` `shift` ``

File: wiki/Help_Center/en.md [hunk 8]
```diff
@@ -440,43 +408,37 @@ If game installer throws errors when attempting to install, you can try few thin
...
 
 Once a pack has been downloaded, extract the .rar into your osu! Songs directory. If you don't know what .rar file is, or how to extract it, you can install free software called 7-zip, after installing it, right click on a beatmap pack file, go to **7-zip** tab, and select "**Extract to 'Beatmap Pack \#xxx**". After you extract the package, you can either double click every single song so osu! opens them, or you can select all songs and drag and drop them into osu! window. Do **not** extract the zip's/osz's yourself, or the beatmaps will display incorrectly in osu and will not function properly.
 
 ------------------------------------------------------------------------
 
-I can hear osu! after launching it, but it's invisible!
--------------------------------------------------------
+### I can hear osu! after launching it, but it's invisible!
 
 Hold **Shift** while starting osu!, and try changing around the settings. Most computers will solve this problem if you use DirectX, vsync and windowed mode, but this may vary from PC to PC.
```
Comment (MegaApplePi) [inline wiki/Help_Center/en.md]: change `**shift**` to `` `Shift` ``

File: wiki/Help_Center/en.md (hunk 2 above)
Comment (TPGPL) [inline wiki/Help_Center/en.md]: probably not

File: wiki/Help_Center/en.md (hunk 6 above)
Comment (TPGPL) [inline wiki/Help_Center/en.md]: code blocks or just code? Code blocks cause line-breaks.

**Pull request merged**